src/Features/CSharp/Portable/CodeRefactorings/MoveDeclarationNearReference/MoveDeclarationNearReferenceCodeRefactoringProvider.cs

```diff
@@ -132,6 +132,7 @@ private StatementSyntax CreateMergedDeclarationStatement(State state, StatementS
         {
             var assignExpression = (AssignmentExpressionSyntax)((ExpressionStatementSyntax)statementSyntax).Expression;
             return SyntaxFactory.LocalDeclarationStatement(
```

Comment:
How could Features use an internal factory?

---
In reply to: [68425672](https://github.com/dotnet/roslyn/pull/12104#discussion_r68425672) [](ancestors = 68425672)